.travis.yml

```diff
@@ -28,15 +28,36 @@ before_install:
         conda config --set always_yes yes --set changeps1 no
         conda update -q conda
         conda info -a
-        conda create -q -n p4env python=$TRAVIS_PYTHON_VERSION ci-psi4 psi4 numpy matplotlib jupyter scipy -c psi4/label/dev -c defaults -c conda-forge
+        conda create -q -n p4env python=$TRAVIS_PYTHON_VERSION ci-psi4 psi4 numpy matplotlib jupyter scipy cython -c psi4/label/dev -c defaults -c conda-forge -c anaconda
 
         source activate p4env
         conda install pytest pytest-cov codecov -c conda-forge
         pip install pytest-shutil
+        pip install git+https://github.com/theochem/iodata.git
```

Comment:
@shivupa This is the line that would trigger installation of Horton 3's iodata package.

Reply:
According to https://github.com/theochem/iodata we can also do
```bash
conda install -c theochem iodata
```
but I'm not sure if we want the one from Anaconda or the master branch. Maybe this is fine.

Reply:
@berquist From what I have tested, `iodata` on conda is old and the name of the attributes have changed since when the latest conda package was built. I think that the `pip` install line can be pegged into a specific commit though, if that fits better.